plugins/field-colour/README.md

```diff
@@ -64,12 +72,55 @@ Blockly.defineBlocksWithJsonArray([
 ]);
 ```
 
+### Blocks
+
+This package also provides four blocks related to the colour field. Each block has generators in JavaScript, Python, PHP, Lua, and Dart.
+- "colour_blend" takes in two colours and a ratio and outputs a single colour.
+- "colour_picker" is a simple block with just the colour field and an output.
+- "colour_random" generates a random colour.
+- "colour_rgb" generates a colour based on red, green, and blue values.
+
+You can install all four blocks by calling `FieldColour.installAllBlocks()`. This will install the blocks and all of their dependencies, including the colour field. You can choose which language generators to install by adding or removing items in the map of generators passed to `installAllBlocks`.
+
+```js
+import { javascriptGenerator } from 'blockly/javascript';
+import { dartGenerator } from 'blockly/dart';
+import { phpGenerator } from 'blockly/php';
+import { pythonGenerator } from 'blockly/python';
+import { luaGenerator } from 'blockly/lua';
+import { FieldColour } from '@blockly/field-colour';
+
+// Installs all four blocks, the colour field, and all of the language generators.
+FieldColour.installAllBlocks({
```

Comment:
It appears you import the `FieldColour` constructor and then try to call a (static) method on it called `installAllBlocks`, which I don't think exists.